cmd/nginx-ingress/main.go

```diff
@@ -410,6 +400,25 @@ func main() {
 	} else {
 		nginxManager = nginx.NewLocalManager("/etc/nginx/", nginxBinaryPath, managerCollector, parseReloadTimeout(*appProtect, *nginxReloadTimeout))
 	}
+	nginxVersion := nginxManager.Version()
+	isPlus := strings.Contains(nginxVersion, "plus")
+	glog.Infof("Using %s", nginxVersion)
+
+	if *nginxPlus && !isPlus {
```

Comment:
Amazing! This has caught me out when I started KIC and I've seen it reported in slack.